src/modules/filelist/File.jsx

```diff
@@ -47,7 +47,7 @@ const File = ({
   onFolderOpen,
   onFileOpen,
   disableSelection = false,
-  isExternalDrive
+  isSharedDrives
```

Comment:
Why rename external by shared? As far as I'm concerned, the two concepts are similar but different. For example, we'll have a different icon for displaying shared drives in the tabs view

Reply:
it was to match the doctype `io.cozy.files.shared-drives-dir`, what do you think?

Reply:
I understand your change better, but I'd put more nuance into it. I would keep the external naming for everything related to the external link. For the moment, we only retrieve external links via the magic folder but soon we'll have folders for shared drives with a special design.

Reply:
For example, I will still use the external here

Reply:
agreed